CertificatePinningTests/TrustKitCertificatePinningTests.swift

```diff
@@ -32,7 +32,7 @@ class TrustKitCertificatePinningTests: XCTestCase {
     /// Communitation between website should work
     func testIfWebsiteWithNotValidCertificateNotWorking() {
```

Comment:
I would say `testIfWebsiteWithoutPredefinedCertificateWorking` name is more valid here - StackOverlfow does not have **any** certificate defined and not just the fact that it's not valid :D

Reply:
and below the other way around 👇 :)